week-3/implement/get-angle-type.js

```diff
@@ -21,3 +21,33 @@
 // Identify Reflex Angles:
 // When the angle is greater than 180 degrees and less than 360 degrees,
 // Then the function should return "Reflex angle"
+
+
+function getAngleType(angle){
+    if (angle === 90){
+        return "Right angle";
+    } else if (angle != 90 && angle < 90){
```

Comment:
this can easily be refactored into angle < 90`. you didnt take care of negative numbers, 360 and numbers greater than 360